tests/python/gpu/test_operator_gpu.py

```diff
@@ -5,6 +5,7 @@
 from test_operator import *
 from test_optimizer import *
 from test_random import *
+from test_sparse_operator import test_sparse_dot
```

Comment:
For some reason it did pass the tests, I noticed when I implemented the cast_storage op and fixed it here https://github.com/dmlc/mxnet/pull/7081.